lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -322,7 +320,6 @@ sub copy_db {
     my ($request) = @_;
     my ($reauth, $database) = _get_database($request);
     return $reauth if $reauth;
-
```

Comment:
From a readability perspective Id like to keep the blank line after the returns. Helps to visually identify them